app/controllers/api/stages_controller.rb

```diff
@@ -19,10 +18,25 @@ def clone
     end
   end
 
+  def update
+    stage.deploy_groups = deploy_groups
+    stage.save!
+    head :no_content
+  end
+
+  def template_stage
+    render json: current_project.template
```

Comment:
I think we want that / it's the normal response for 'not found' :D

On Tue, Aug 16, 2016 at 5:28 PM, Adam Panzer <notifications@github.com>
wrote:

> In app/controllers/api/stages_controller.rb
> <https://github.com/zendesk/samson/pull/1198#discussion_r75045355>:
>
> > @@ -19,10 +18,25 @@ def clone
> >      end
> >    end
> >
> > +  def update
> > +    stage.deploy_groups = deploy_groups
> > +    stage.save!
> > +    head :no_content
> > +  end
> > +
> > +  def template_stage
> > +    render json: current_project.template
>
> 404 will raise in an api client, so we don't necessarily want to do
> that... right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1198/files/66abc156d3f3ad8478ca07ee5c0e76ed494978bf#r75045355>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ2Kz21UUA9sCrI9gKXW_8h7-cSBlks5qglWRgaJpZM4Jhtog>
> .
>